arcade-drive-xbox-controller/robot.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+"""
+    This is a demo program showing the use of the DifferentialDrive class.
+    Runs the motors with split arcade steering and an Xbox controller.
+"""
+
+import wpilib
+import wpilib.drive
+
+
+class MyRobot(wpilib.TimedRobot):
+    def robotInit(self):
+        """Robot initialization function"""
+
+        # create motor controller objects
+        m_left = wpilib.Talon(0)
```

Comment:
I would not prefix these with m_

Also, https://github.com/wpilibsuite/allwpilib/blob/main/wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/arcadedrivexboxcontroller/Robot.java has these as `PWMSparkMax` motors.